.github/workflows/ci.yml

```diff
@@ -3,27 +3,86 @@ on:
   - push
   - pull_request
 
+env:
+  GO_VERSION: "1.17"
+
 jobs:
+  lint:
+    name: Lint
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          args: >-
+            --verbose
+            --timeout=5m
```

Comment:
We should probably just plop this in the config file

Reply:
That's why I just run it before pushing, if it even got close to the default of 1 I'd be livid :) haha

Reply:
The first run always takes a bit, but it takes like 1 or 2 seconds, which is still kinda long, but ...